homework_5/demo/src/main/java/com/epam/demo/model/User.java

```diff
@@ -1,95 +1,48 @@
 package com.epam.demo.model;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
 
-/**
- * Class that describes user.
- *
- * @author Vladislav Prokopenko.
- */
 @Entity
 @Table(name = "user")
+@ToString(exclude = "password")
 public class User {
 
     @Id
+    @Getter
+    @Setter
```

Comment:
You can put `@Getter` and `@Setter` on class level and don't put it above each field